### PR TITLE
feat: ITV-39 add refuse and recyling step

### PR DESCRIPTION
**Pull request by mrmartineau**
ITV-39 add refuse step

# What?

- Added new refuse and recyling sub-step
- fixed a React issue with the antisocial behaviour step (pretty sure it was my mistake from #19)

image

### CODE REVIEW

File: steps/property-inspection/refuse.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "refuse-notes",
+          Component: PostVisitActionInput,
+          props: {
+            label: {
+              value: (
+                <>
+                  Add note about antisocial behaviour <b>by</b> or{" "}
```
Comment (LBHABEBIC) [inline steps/property-inspection/refuse.tsx]: Looks like the wrong note above the box

**Review by LBHABEBIC — COMMENTED** [review]
see comment about note. everything else ok.

File: steps/property-inspection/refuse.tsx (hunk 1 above)
Comment (mrmartineau) [reply steps/property-inspection/refuse.tsx]: thanks, good catch. It is fixed now.

**Review by LBH-wgreeff — APPROVED** [review]
looks good

**Pull request merged**